Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### 1. Describe some of the differences between `.forEach` & `.map`.
```

Comment:
I'm worried that your understanding might not be 100% on point for these array functions